igvc_navigation/launch/follower.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
I should add that, even with these launch parameters, I still observe the jerky behavior mentioned in my[ first comment ](https://github.com/RoboJackets/igvc-software/pull/373#issuecomment-476398552) for this PR.